clients/TypeScript/packages/client/src/ChainSync/ChainSyncClient.ts

```diff
@@ -12,29 +11,33 @@ import { requestNext } from './requestNext'
 
 export interface ChainSyncClient {
   context: InteractionContext
-  requestNext: (options?: { mirror?: Mirror }) => void
```

Comment:
So this means that the chain-sync is necessarily pipelined?

Reply:
Actually no, because one can always specify a `requestBuffer: 1` to get a one-by-one behavior (if that's even a useful use-case :thinking: ?)